docs/readme-zephyr.md

```diff
@@ -163,51 +163,53 @@ example) can be used to sign images.
 
 ## Serial recovery
 
-### interface selection
+### Interface selection
 
-Serial recovery protocol can be availeble over either the serial port or the USB CDC ACM virtual serial port.
+A serial recovery protocol is available over either a serial port or an USB CDC ACM virtual serial port.
 The SMP server implementation can be enabled by ``CONFIG_MCUBOOT_SERIAL=y``.
-Kind of interface used can be set using ``BOOT_SERIAL_DEVICE`` Kconfig chose: ``CONFIG_BOOT_SERIAL_UART`` or ``CONFIG_BOOT_SERIAL_CDC_ACM`` can be selected.
-Which interface belongs to the protocol shall be set by the devicetree-chosen-node:
+To set a type of an interface, use ``BOOT_SERIAL_DEVICE`` Kconfig option and choose either ``CONFIG_BOOT_SERIAL_UART`` or ``CONFIG_BOOT_SERIAL_CDC_ACM``.
+Which interface belongs to the protocol shall be set by the devicetree-chosen node:
 - `zephyr,console` if serial port used.
 - `zephyr,cdc-acm-uart` if virtual serial port used.
 
 ### Entering to the serial recovery mode
 
-Mode can be entered once the device gets reboots and triggering event happens.
+The serial recovery mode can be entered once the device is rebooted or powered up and event is triggered
```

Comment:
The serial recovery mode can be entered once the device is rebooted and a triggering event happens.

Reply:
```suggestion
To enter the serial recovery mode, the device has to initiate rebooting, and a triggering event has to occur, for example, pressing a button.
```
As discussed on the chat.

Reply:
Corrected as discussed on the chat.